### ISSUE TITLE
missing sources, buildme.sh and build logs

### ISSUE THREAD

**Issue by klslz** [LMS-Community/plugin-LocalPlayer#1]
I can't find the sources, buildme.sh (build scripts/processes ) and build logs for the supplied squeezelite binaries. 

If these data don't exist, the binaries would have to be replaced and the build data and sources would have to be added.

There are rather new official squeezelite binaries on sourceforge. However, from my point of view, these wouldn't 
comply to the usual license terms either.

This plugin would require the same setup as done for the LMS 3pp binaries (flac,sox,etc).

**Comment by terual** [comment]
This repository is just a copy of the original plugin by Triode. Feel free to submit a pull request!

**Comment by michaelherger** [comment]
@terual are the "releases" just copies, too? I was wondering whether I should tweak the plugin to not install/initialize on pCP, as it conflicts with pCP's core feature. But I wouldn't know how to build a release.

**Comment by terual** [comment]
Yeah, the releases are copies also. When Triodes repo was whut down I downloaded all the release zips. To create this repo I extracted the zips into a this repo and made a commit after each zip extraction to "artificially" create a commit history. I have never made a new release, but it shouldn't be to hard as everything is contained in this repo.

**Comment by terual** [comment]
Is this the only plugin which conflicts with pCP? If there are more we could maybe add a compatibility flag to the repo xml format which pCP could use?

**Comment by michaelherger** [comment]
> Is this the only plugin which conflicts with pCP? If there are more we could maybe add a compatibility flag to the repo xml format which pCP could use?

I don't know. And the actual issue that was reported sounded really wrong (LMS starting multiple times). I only thought this plugin didn't make sense in a pCP environment, because pCP _is_ about being a player from the ground up. So there's no need to install this plugin.

But as this was the first time I heard about this issue, I believe we don't really need to do anything anyway.
